content/en/status.md

```diff
@@ -9,19 +9,14 @@ description: Maturity-level of the main OpenTelemetry components
 
 ## {{% param title %}}
 
-The development status, or maturity level, of the [main OpenTelemetry
-components][main-comp] are as follows:
-
+OpenTelemetry is made up of several components, each having different meanings
+for "status". For example, the status of a signal in the specification may not
+be the same as its status in a particular language SDK.
```

Comment:
Oh! I don't really like the phrase _each having different meanings for "status"._

Hmm, after think about this for a bit, ... how about:

```suggestion
OpenTelemetry is made up of [several components][main-comp], some
language-specific and others language-agnostic. When looking for a status, make
sure to look for the status from the right component page. For example, the
status of a signal in the specification may not be the same as the signal status
in a particular language SDK.
```